crates/binstalk-types/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "binstalk-types"
+description = "The binstall toolkit that contains basic types to cooperate with binstalk-manifests"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/binstalk-manifests-types"
```

Comment:
Link needs changing

also suggest description change as it's no longer just a manifest subcrate

Reply:
Done!